src/Particular.ThroughputCollector.Persistence.RavenDb/ThroughputDataStore.cs

```diff
@@ -3,10 +3,10 @@
 using Contracts;
 using Particular.ThroughputCollector.Persistence.RavenDb.Models;
 using Raven.Client.Documents;
+using Raven.Client.Documents.Linq;
 
 class ThroughputDataStore(
     IDocumentStore store,
-    PersistenceSettings persistenceSettings,
     DatabaseConfiguration databaseConfiguration) : IThroughputDataStore
 {
     const string ThroughputTimeSeriesName = "INC: throughput data";
```

Comment:
Do we need to make this configurable, in case of conflicts with other timeseries?